### DISCUSSION TITLE
Welcome to AxMath/AxGlyph/AxCells Discussions!

### DISCUSSION THREAD

**Discussion by loongmxbt** [loongmxbt/AxMath#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MaybeGoodman** [comment]
关于axglyph希望有两点更新：
1，.3D图样式虽然用起来很方便，但是样式太少了，尤其是斜棱锥和棱锥的调节，试了很久也没找到控制点
2.关于物理的示意图太少了，针对初高中会用到大量的电磁铁等画示意图，由于库里面没有，手动画十分不便。

**Reply by nairaalamvt** [reply]
I also hope, we have the ability to create or add custom symbols that we can reuse. I know the software is very cheap (for some people) and the developer might not be making a lot of money from it, but I see this as a direction in which the software could grow very much.

**Comment by lawnwezza** [comment]
I bought a AxMath and my computer that I installed it in crashed. It couldn't use the license key for my new computer. As such I had to buy another license. I have just reinstalled my computer and wanted to retrieve my license only to discover that my account had no trace of my previous purchases. It's very unfair if I have to buy a software license every time that my computer has an issue. I have attached the receipt of my previous purchase to see if this license can be restored to me as soon as possible. Thank you
![axsoft](https://github.com/user-attachments/assets/f0fd6f3e-735e-4bb5-8fcb-b4fad0b1eaad)

**Reply by loongmxbt** [reply]
Hi, you can send us an email to get a new license.

**Reply by lawnwezza** [reply]
Thank you so much! My email is law_nwezza@yahoo.com

**Comment by lawnwezza** [comment]
My email address is ***@***.*** 
Thank you. 
Lawrence. 

Yahoo Mail: Search, organise, conquer 
 
  On Wed, 25 Dec 2024 at 12:07, Saint ***@***.***> wrote:   


Hi, you can send us an email to get a new license.

**Comment by SonAbhi-66** [comment]
Hi 
I recently purchased the AxSoft license for Axglyph, but I have not received a license file or key. Can you help me with this? Here is receipt ID - #1968-1523

**Reply by loongmxbt** [reply]
Hi, we've resend your license.

**Reply by SonAbhi-66** [reply]
Thank you! I got it. Can you send me the license file for another purchase I made a while ago, here is the receipt number, [#1624-3216], I realized, I never got the license file for this.

**Comment by EmanuelNich** [comment]
I bought Axglyph, Axcell and Labplot for $33.00 only Axglyph was downloaded and the other two just varnsihed. My previous license is not working and no one answers. Any help....

**Comment by AnDaraDoras** [comment]
Hi
I have problems with AxMath add-in - it is reassigning all my normal fonts (e.g. Ctrl F becomes Fonts) and is preventing mouse scroll. I decided to re-install. I have version 20241001_EN. When it was installed it informed me there was an update ([Win_20250218_EN]) which I downloaded. However this was flagged as Trojan by MS Defender and deleted from my system.  This problem has already be flagged - see https://github.com/loongmxbt/AxMath/issues/30. will this be fixed?

**Comment by thanh36c11913** [comment]
Xin chào,
tôi vừa mua bản quyền AxSoft cho Axglyph, nhưng tôi chưa nhận được tệp bản quyền hoặc khóa bản quyền. Bạn có thể giúp tôi không? Đây là ID biên lai.
![178d9a14-a59b-47f8-84c7-4b3cf5b88373](https://github.com/user-attachments/assets/97e00fb6-8b2a-436a-94c4-7fafea7cfad7)

**Comment by igorbogdanoski** [comment]
hello.
 I have AxMath and AxGluph.
Now there is a new updated version. In order to upgrade I need to download the new version. I will need a new license, or I can use my old one. I have 2.7061 build 20250218

**Comment by levuphuc1587** [comment]
Tôi đã mua máy tính mới và muốn chuyển ứng dụng sang máy mới để tiếp tục làm việc nhưng giấy phép cũ lại cảnh báo không hợp lệ và không có giao dịch nào được lưu lại và tôi không tìm được cách khôi phục bản quyền của phần mềm này . Thực sự bất công nếu tôi phải mua bản quyền phần mềm mỗi khi muốn cài đặt một máy tính khác. Tôi đã đính kèm biên lai mua hàng trước đó để xem tài liệu bản quyền này có thể được khôi phục cho tôi sớm nhất có thể hay không. Cảm ơn bạn.
<img width="734" height="739" alt="unnamed" src="https://github.com/user-attachments/assets/05ad3446-1bab-4105-9ad8-21d6725aa363" />
